### PR TITLE
[PERFSCALE-2485]: Adds ROSA 4.14 Control Plane tests

### PR DESCRIPTION
**Pull request by krishvoor**
This PR adds control-plane tests for ROSA 4.14 nightly

### CODE REVIEW

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-perfscale-rosa-multiaz-control-plane-24nodes-periodic

**Review by afcollins — APPROVED** [review]
LGTM

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 1]
```diff
@@ -3436,3 +3436,154 @@ periodics:
...
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale-qe
+    ci-operator.openshift.io/variant: rosa-4.14-nightly-x86
+    ci.openshift.io/generator: prowgen
+    job-release: "4.14"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-perfscale-rosa-multiaz-node-density-heavy-24nodes-periodic
+  spec:
```
Comment (paigerube14) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:3533]: let's add the reporter_config section under the name of these new tests so we can track the results in slack

```
reporter_config:
    slack:
      channel: '#ocp-qe-scale-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :white_check_mark: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :white_check_mark:
        {{else}} :warning:  Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> :warning: {{end}}'
```

Comment (krishvoor) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:3533]: Sure, adding this!

Comment (krishvoor) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:3555]: Added the reporter_config.
Closing this.

**Pull request merged**